src/logging/LoggedComand.java

```diff
@@ -0,0 +1,32 @@
+package logging;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class LoggedComand extends Command {
+
+    public static boolean IS_LOG = false;
```

Comment:
Create a LogManager so this parameter would be configurable.

Reply:
Would it be enough if I instead made a static method 
enableLogs(boolean enable)
and 
isLoggingEnagled()

Reply:
done